metadata-ingestion/src/datahub/ingestion/source/mlflow.py

```diff
@@ -595,12 +598,25 @@ def _traverse_mlflow_search_func(
         Utility to traverse an MLflow search_* functions which return PagedList.
         """
         next_page_token = None
-        while True:
-            paged_list = search_func(page_token=next_page_token, **kwargs)
-            yield from paged_list.to_list()
-            next_page_token = paged_list.token
-            if not next_page_token:
+        try:
+            while True:
+                paged_list = search_func(page_token=next_page_token, **kwargs)
+                yield from paged_list.to_list()
+                next_page_token = paged_list.token
+                if not next_page_token:
+                    return
+        except Exception as e:
```

Comment:
i've noticed that if we don't catch here, it will raise pipeline error so the whole ingestion fails.
I've added a logger as a workaround since this is defined as static method, but would love to get any advice on logging here.